web_widget_text_markdown/static/src/css/web_widget_text_markdown.css

```diff
@@ -0,0 +1,7 @@
+.o_field_text_markdown blockquote {
+    background: #f9f9f9;
+    border-left: 10px solid #ccc;
+    margin: 1.5em 10px;
+    padding: 1em 10px 0.1em 10px;
+    quotes: "\201C""\201D""\2018""\2019";
+}
```

Comment:
Am not really for this 8.0 css on v13.0, since we are using showdown js and also bootstrap css to render some elements to hmtl using the css classes. But just added it for you to test out,

Reply:
Ok I get it. Let's declare this finished when:

- stuff we are adding is SCSS not CSS and gets compiled into the assets bundle like normal
- everything looks great and roughly the way on `project.task` like the way it was on 8.0 (so no loads of whitespace)